dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Admin/UserImport.cshtml.cs

```diff
@@ -84,7 +96,8 @@ public async Task<IActionResult> OnGetDownloadFile()
                     RowNumber = r.RowNumber,
                     Id = r.Id,
                     UserId = r.UserId,
-                    Errors = r.Errors is null ? null : string.Join(".", r.Errors),
+                    UserImportRowResult = r.UserImportRowResult,
+                    Notes = r.Notes is null ? null : string.Join(".", r.Notes),
```

Comment:
Just noticed this...
```suggestion
                    Notes = r.Notes is null ? null : string.Join(". ", r.Notes),
```

Reply:
Done